reapers/policies.go

```diff
@@ -207,7 +227,7 @@ func (p *PoliciesReaper) handlePolicyEvent(logger *zap.Logger, keyName string, e
 	}
 
 	if event.Typ == kvstore.EventTypeCreate {
-		logger.Debug("Creating policy", zap.Strings("labels", labels.GetModel()))
+		logger.Info("Creating policy", zap.Strings("labels", labels.GetModel()))
```

Comment:
Joey switched netreap to turn off debug logging because these messages were filling up their aws logs with too many. (cause its 60k + per server). Not sure if we want to switch this to Info.